### ISSUE TITLE
Review Design Wireframe: Donation page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#94]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by Oyvindeavor** [comment]
## Desktop
![Screenshot](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/50bb3410-c41a-4bd6-9456-76b68bcb9cfa)

- None of the inputs work.
- Checkbox "every month donation" doesn't work.
- Checkboxes "debit card" and "credit card" don't work.
- The submit button looks a little small, which can be hard to click; increasing the clickable area's height can fix it.

**Comment by lem31** [comment]
It's just the Figma design, so no need to worry about the buttons etc working :-)

**Comment by Oyvindeavor** [comment]
Noted just tought they needed to work since it was a hifi wireframe.  :)

**Comment by Vickabu** [comment]
This issues is not relevandt since prototyping is not a part of the design

**Comment by Oyvindeavor** [comment]
Closing
